contibuting.md

```diff
@@ -26,8 +26,8 @@ git push origin नयी-ब्रांच-का-नाम
 हर पुल्ल रिक्वेस्ट को २ चरण के रिव्यु को पार करना ज़रूरी है। रिव्यु के लिए हम निचे दी गयी बातों का पबातों का पालन करें।
 
 - सरे वर्गों का नाम अंग्रेज़ी में पुल्ल रिक्वेस्ट के विवरण में लिखें। जिसको रीवीउअर रिव्यु पूरा होने के बाद चेक मार्क करेगा।
-- किसी भी रीवीउअर के रीव्यु फीडबैक को रीसोलव मार्क न करें। रीवीउअर आपके फीडबैक फिक्सेस को देखने के बाद उसको रीसोलव मार्क करेगा।
-- आप फीडबैक में परिवर्तन करने के बाद उस फीडबैक पर कमेंट डालें या ये 👍 (इमोजी) लगा दें जिससे रीवीउअर को पता चल सके की आपने इस फीडबैक का फिक्स कर दिया है।
+- किसी भी रीवीउअर के रीव्यु फीडबैक को रीसोलव मार्क ना करें। रीवीउअर आपके फीडबैक फिक्सेस को देखने के बाद उसको रीसोलव मार्क करेगा।
+- आप फीडबैक में परिवर्तन करने के बाद उस फीडबैक पर कमेंट डालें या ये 👍 (इमोजी) लगा दें जिससे रीवीउअर को पता चल सके की आपने इस फीडबैक को फिक्स कर दिया है।
```

Comment:
For this, `का` will be better I guess.
As `फीडबैक` is not a issue, feedback is telling about a issue. So `फीडबैक का फिक्स कर दिया`  will be right here I guess.
Any thoughts?

Reply:
`For this, का will be better I guess.` as per your thought it would be better. :+1:  I'm gonna change it.